portal/portal-impl/impl/src/java/org/sakaiproject/portal/entityprovider/PortalEntityProvider.java

```diff
@@ -18,23 +23,54 @@
 import org.sakaiproject.entitybroker.EntityReference;
 import org.sakaiproject.entitybroker.EntityView;
 import org.sakaiproject.entitybroker.entityprovider.annotations.EntityCustomAction;
+import org.sakaiproject.entitybroker.entityprovider.extension.ActionReturn;
 import org.sakaiproject.entitybroker.entityprovider.extension.Formats;
 
 import org.sakaiproject.tool.api.Session;
 import org.sakaiproject.tool.api.SessionManager;
 
+import org.sakaiproject.portal.api.PortalService;
 import org.sakaiproject.portal.beans.PortalNotifications;
+import org.sakaiproject.portal.beans.BullhornAlert;
+
+import org.sakaiproject.profile2.logic.ProfileConnectionsLogic;
+import org.sakaiproject.profile2.logic.ProfileLinkLogic;
+import org.sakaiproject.profile2.model.Person;
+
+import org.sakaiproject.util.ResourceLoader;
 
 /**
  * An entity provider to serve Portal information
  * 
  */
 public class PortalEntityProvider extends AbstractEntityProvider implements AutoRegisterEntityProvider, Outputable, ActionsExecutable, Describeable {
 
+	private static final Log log = LogFactory.getLog(PortalEntityProvider.class);
+
 	public final static String PREFIX = "portal";
 	public final static String TOOL_ID = "sakai.portal";
 
-	public SessionManager sessionManager;
+	private DeveloperHelperService developerHelperService;
```

Comment:
This is redundant as it's injected in `AbstractEntityProvider` which this extends